test/controllers/api/v2/organizations_controller_test.rb

```diff
@@ -113,7 +113,14 @@ class Api::V2::OrganizationsControllerTest < ActionController::TestCase
   end
 
   test "should not create with content type text plain" do
-    assert_raises_with_message(RuntimeError, 'Unknown Content-Type') do
+    if Rails.version.start_with?('5.2')
+      error_type = RuntimeError
+      error_message = 'Unknown Content-Type'
+    else
+      error_type = InvalidMimeType
+      error_message = '"" is not a valid MIME type'
```

Comment:
IMHO the correct response should be `text/plain is not a valid MIME type`.

Reply:
actually, the more i look at it this looks like the test isn't testing what it should. 
We should not be raising this error at all, we should display a nice error message in this case:
https://github.com/theforeman/foreman/blob/c782cd61254ff11423dc2d09a627c070d007de65/app/controllers/api/v2/base_controller.rb#L146
but i guess this error gets thrown before the callback

Reply:
now should be fixed to test this correctly, in a way that doesn't care about rails version.